js/prototype/validation.js

```diff
@@ -916,6 +916,25 @@ function parseNumber(v)
     return parseFloat(v);
 }
 
+function validateCreditCard(input) {
```

Comment:
Test 
`validateCreditCard('')` returns `true`. 
`validateCreditCard('222 2222')` returns `true`. 

Need some tweaking.

Reply:
@kiatng did you run the same tests on the old version?

<img width="1147" alt="Screenshot 2024-05-20 alle 11 15 30" src="https://github.com/OpenMage/magento-lts/assets/909743/f3edc9e7-6bf9-4a91-80cd-90585b5b45af">